waveorder/stokes.py

```diff
@@ -0,0 +1,344 @@
+"""
+Overview
+--------
+
+This module collects Stokes- and Mueller-related calculations. 
+
+The functions are organized into three groups:
+
+1) A forward function group: 
+
+A = A_matrix(swing, scheme="5-State")
+s0, s1, s3, s3 = s0123_CPL_after_ADR(ret, ori, tra, dop)
+s0, s1, s2 = s012_CPL_after_AR(ret, ori, dop)
```

Comment:
These names can read like a jargon. How about `stokes_ADR_CPL` and `stokes_AR_CPL`. Writing down the mnemonics for components in the order in which their Mueller matrices multiply can make it easier to discover the methods if we add more transformations.

Reply:
I think adding `input="CPL"` is a big improvement!

How do these sound?
`stokes_after_ADR(ret,ori,tra,dop, input='CPL')`
`stokes012_after_AR(ret,ori,tra, input='CPL')`

I think it's important to make it clear that these are stokes vectors _after_ an ADR/AR, and I think if we don't return all 4 stokes params then the function name should make that clear. 

> Reading the mnemonics for components in the order in which vectors and matrices multiply is more intuitive. The input can also be a Stokes vector when one wants to use the methods for another purpose, e.g., model non-ideal illumination.

I'm not sure I understand this. The matched `inverse` functions below take Stokes vectors as input...is this what you have in mind?